Sample issue templates (#4)
algolia-samples/about: .github/ISSUE_TEMPLATE/code-submission.md

---

> name: Sample Code Submission

Reviewer: ```suggestion
name: Sample code submission
```

> about: Submit a sample to be added to the Algolia Code Exchange

Reviewer: ```suggestion
about: Submit a sample for the Algolia Code Exchange
```

> title: '[Code Sample Submission]'

Reviewer: ```suggestion
title: '[Sample code submission]'
```

labels: code-submission
assignees: 'clemfromspace'
---

**What's the use case for the sample?**

> Please explain what use case this sample is solving. If this is another programming language implementation for an existing sample, please copy the link here.

Reviewer: ```suggestion
Please explain what use case this sample is addressing. 
If your sample is an implementation of an existing sample in another programming language, please add a link to the existing sample here.
```

**What's the programming language?**

> What programming language is primarily used for this template? If there's a front-end and back-end component, please list both.

Reviewer: ```suggestion
What's the primary programming language for this sample? If your sample has both front-end and back-end components, please list both.
```

**Sample GitHub URL:**

> Please list the GitHub URL of your project. e.g. github.com/algolia-samples/chatbot-with-algolia-answers

Reviewer: ```suggestion
Please list the GitHub URL of your project, for example, `github.com/algolia-samples/chatbot-with-algolia-answers`
```

**Author Information (please fill out):**

- Author name: [e.g. Peter Parker]
- Author GitHub Handle: [e.g. github.com/peterparker]
- Organization name (if submitted on behalf of a company): [e.g. Algolia]
- Organization GitHub URL(if submitted on behalf of a company): [e.g. github.com/algolia]

**Dependencies:**

> Please list any dependencies for services, tools or similar this project has (including Algolia's one). e.g: Algolia Answers, Google Cloud Vision, etc.

Reviewer: ```suggestion
Please list any dependencies for services, tools, or others this project has, including Algolia's services. For example, Algolia Answers, Google Cloud Vision, etc.
```
